data/json/recipes/armor/boots.json

```diff
@@ -16,6 +16,23 @@
     ]
   },
   {
+    "result": "boots_chitin",
+    "type": "recipe",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_FEET",
+    "skill_used": "tailor",
+    "difficulty": 6,
```

Comment:
It kinda is, because it makes it not show up in the recipe list, making it easy to forget.
And chitin won't drop until that lvl 3 in survival anyway. It has really strict requirements at the moment, though it's mostly due to lack of bigger chitinous critters.
Survival 3 is already a breakpoint for CBMs, it doesn't need more roles.